src/components/button/README.md

```diff
@@ -2,14 +2,50 @@
 
 ## Introduction
 
-A button is an element that allows users to carry out an action on a GOV.UK page. Common use cases include allowing a user to **Start** an application or **Save and continue** their progress. A button should have a short text snippet that describes what it will do:
+A button is an element that allows users to carry out an action on a GOV.UK page. Common use cases include allowing a user to **Start** an application or **Save and continue** their progress. A button should have a short text snippet that describes what it will do.
 
 [Preview the button component.](http://govuk-frontend-review.herokuapp.com/components/button/preview)
 
 ## Guidance
 
 More information about when to use button can be found on [GOV.UK Design System](http://www.linktodesignsystem.com/button "Link to read guidance on the use of button on Gov.uk Design system website")
 
+## Quick start examples
+
+Buttons are configured to perform an action and they can have a different look. For example, they can be disabled until a valid action has been performed by the user.
+
+    <input class="govuk-c-button  " value="Save and continue" >
+
+## Variants
+
+### Button--disabled
+
+[Preview button--disabled variant.](/components/button/button--disabled/preview)
+
+Markup
+
+    <input class="govuk-c-button  " value="Save and continue" >
```

Comment:
This is the same markup as above – it's not disabled…

Also, possibly a bit pedantic, but it'd also be good to try and remove the extra spaces in the classes attribute.